keg_auth/model/__init__.py

```diff
@@ -490,8 +490,8 @@ class BundleMixin(object):
     @might_commit
     @might_flush
     @classmethod
-    def edit(cls, oid=None, **kwargs):
```

Comment:
What's the reason behind this change? It doesn't seem to be relevant to the issue, and the generic keg-elements method has `oid` (https://github.com/level12/keg-elements/blob/master/keg_elements/db/mixins.py#L195).

Reply:
This was a change I made to get the tests passing because `oid` was throwing back an "Unknown column or relationship" error. That made sense to me because I knew the column should be id, so I fixed it without investigating further, which I should not have done.

I've done a bit of digging, and it seems there's a small bug in KE. The `@kwargs_match_entity` decorator used on the `edit` method on the `MethodsMixin` (https://github.com/level12/keg-elements/blob/168e70af2b49bfd3f1e97705b86d158821e5d2ad/keg_elements/db/mixins.py#L193) will throw an error when `oid` is passed in as a kwarg (because it's not a column or relationship on the entity). We don't catch this in the tests because we pass the `oid` in without the keyword: https://github.com/level12/keg-elements/blob/168e70af2b49bfd3f1e97705b86d158821e5d2ad/keg_elements/tests/test_db_mixins.py#L77.

I created a pr in KE (https://github.com/level12/keg-elements/pull/175) and pushed up a commit to this pr with a better workaround.

Reply:
@calebsyring your last commit is the correct fix for this issue.

I believe this error started cropping up in tests due to a change @guruofgentoo made a few months ago to KE: https://github.com/level12/keg-elements/blame/168e70af2b49bfd3f1e97705b86d158821e5d2ad/keg_elements/db/mixins.py#L193